dev-itpro/developer/methods-auto/date/date-year-method.md

```diff
@@ -34,6 +34,11 @@ The year.
 
 
 [//]: # (IMPORTANT: END>DO_NOT_EDIT)
+## Remarks
+Don't use this method to replace [Date2DWY(Date, 3)](../system/system-date2dwy-method).
```

Comment:
```suggestion
Don't use this method to replace [`Date2DWY(Date, 3)]`(../system/system-date2dwy-method.md).
```

Reply:
I've added the `.md` extension.
If I format the text as code then in my local preview the link isn't displayed as a clickable link. 
Therefore I don't want to add the `code` formatting here

Reply:
In my original suggestion, I made a typo. I think this is the one you tested. The current suggestion does work in the preview.

Reply:
If I uderstand you correctly then you woud suggest the following change?
```md
[`Date2DWY(Date, 3)`](../system/system-date2dwy-method.md)

```
In GitHub this is rendered correctly (could be yn issue with my local VS Code Extension)
[`Date2DWY(Date, 3)`](https://github.com/MicrosoftDocs/dynamics365smb-devitpro-pb/pull/3457)
I will update my code with the inline code formatting.

Reply:
Yes, that's right. 
(Btw, I usually edit the docs on GitHub only. In rare cases, I expand github.com to github.dev).